network/common/src/main/java/org/apache/spark/network/server/StreamManager.java

```diff
@@ -47,6 +47,14 @@
   public abstract ManagedBuffer getChunk(long streamId, int chunkIndex);
 
   /**
+   * Called in response to a stream() request. The returned data is streamed to the client
+   * through a single TCP connection.
+   */
+  public ManagedBuffer openStream(String streamId) {
```

Comment:
I wonder if we should use another name for `streamId`, to distinguish it from the `long streamId` in the other methods here, to avoid confusion.  Even `openStream` could be confused since the existing code also talks about streams a lot.  "unframedStream" is the best I can think of.